CHANGES.md

```diff
@@ -1,5 +1,119 @@
 # Changes
 
+## stable-2.9.0
+
+This release extends Linkerd's zero-config mutual TLS (mTLS) support to all TCP
+connections, allowing Linkerd to transparently encrypt and authenticate all TCP
+connections in the cluster the moment it's installed. It also adds ARM support,
+introduces a new multi-core proxy runtime for higher throughput, adds support
+for Kubernetes service topologies, and lots, lots more, as described below:
+
+* Proxy
+  * Performed internal improvements for lower latencies under high concurrency
+  * Reduced performance impact of logging , especially when the `debug` or
+    `trace` log levels are disabled
+  * Improved error handling for DNS errors encountered when discovering control
+    plane addresses, which can be common during installation, before all
+    components have been started, allowing linkerd to continue to operate
+    normally in HA during node outages
```

Comment:
Nit: I feel like this reads a little odd with all the commas, maybe something like:

```suggestion
  * Improved error handling for DNS errors encountered when discovering control
    plane addresses; this can be common during installation before all
    components have been started, allowing linkerd to continue to operate
    normally in HA during node outages
```

Reply:
I'm also a big semicolon fan :slightly_smiling_face: